### DISCUSSION TITLE
Possibility to map a feed in a config file (not through admin)

### DISCUSSION THREAD

**Discussion by stigzelm** [engram-design/FeedMe#923] [Ideas]
### Description
It would be really useful if you could override a feeds mapping through a config.php file. The same way we are overriding siteUrl in the general.php file.

**Comment by KONEY** [comment]
We have the same need because our feed are pretty big and take quite sometime to process. So to change a simple field mapping it takes some twenty minutes.

Alternatively some documentation on how settings are saved would be helpful, at this point reading the content of feedme_feeds table, fieldMapping is quite confusing.

**Comment by ben-at-sahmri** [comment]
This would be a huge help - it would make deploying a feed configuration and working in a shared environment a lot easier and more robust.

**Comment by gbowne-quickbase** [comment]
I'd want to be able to export an existing feed first since setup from scratch as an array or json object could be brutal. But once the pattern is established, it would mean a dev could modify a feed, and include that config with the PR, and when it is pushed up, voilà it'll work as required, instead of rushing to make feedme changes after deployment.
